### PR TITLE
Display the number transaction in Posting Journal

### LINKED ISSUE #1551: Display the number of lines after the filter with the posting journal
Display the number of lines after the filter with the posting journal

### PR DESCRIPTION
**Pull request by lomamech**
Display the number transaction in Footer of Posting Journal

closes #1551

### CODE REVIEW

**Comment by ghost** [comment]
:umbrella: The latest upstream changes (presumably 033d9cd628a4576dac6b6ea28b8b947b370ddbdc) made this pull request unmergeable. Please resolve the merge conflicts.

File: client/src/modules/journal/journal.js [hunk 1]
```diff
@@ -312,14 +322,20 @@ function JournalController(Journal, Sorting, Grouping,
...
       .then(function (records) {
+        // To Get the number of transaction 
+        transactionNumber =  records.aggregate.length;
 
         // pre process data - this should be done in a more generic way in a service
         vm.gridOptions.data = transactions.preprocessJournalData(records);
-
+        vm.gridOptions.showGridFooter = true;
+        vm.gridOptions.gridFooterTemplate = '<div><strong>' + $translate.instant('FORM.INFO.NUM_TRANSACTION') + 
+          ' : ' + transactionNumber + ' / ' + vm.numberTransaction + '</strong></div>';
```
Comment (sfount) [inline client/src/modules/journal/journal.js]: @lomamech can you rename these variables to something that provides more context? There is no real difference between `numberTransaction` and `transactionNumber` - this might be hard to follow when revisiting this code in the future. 

For example: 
```js
// number of all of the transactions in the system
var numberTotalSystemTransactions

// number of transactions downloaded and shown in the current journal 
var numberCurrentGridTransactions
```

**Review by sfount — COMMENTED** [review]
This change looks good to me! if possible could you include a quick screenshot of the feature that is added here to get an idea of how it works along with this PR?

**Comment by jniles** [comment]
LGTM :+1:
